content/en/about/why-falco.md

```diff
@@ -199,7 +199,7 @@ Falco only monitors relevant events by using filters and rules to define which e
 
 ##### Kernel-level instrumentation to capture system events
 
-Falco uses kernel-level instrumentation to capture system events by intercepting system calls and other kernel-level events in real-time. This approach allows Falco to detect and respond to security incidents more quickly and effectively, as it can analyze events before they are processed by the operating system or applications. Additionally, kernel-level instrumentation ensures that Falco is highly performant, as it can intercept and analyze events with minimal overhead.
+Falco uses kernel-level instrumentation to capture system events by intercepting system calls and other kernel-level events in real-time. This approach allows Falco to detect abnormal behaviour and security threats quickly and effectively, as it can analyze events before they are processed by the operating system or applications. Additionally, kernel-level instrumentation ensures that Falco is highly performant, as it can intercept and analyze events with minimal overhead.
```

Comment:
```suggestion
Falco uses kernel-level instrumentation to monitor system events by observing system calls and other kernel-level events in real-time. This non-intrusive approach enables Falco to capture rich application activity signals and to detect and alert about abnormal behavior and security threats as soon as they happen. The events gathered can later be examined in SIEM or data lake systems. Additionally, Falco's developers are committed to maximizing the kernel instrumentation performance and ensuring events monitoring and processing come with minimal overhead.
```

- `capture` :point_right: `monitor` for the same reason explained by @pmusa in [this comment](https://github.com/falcosecurity/falco-website/pull/1012#discussion_r1239361221)
   - I also suggest modifying the title from `Kernel-level instrumentation to capture system events` :point_right: `Kernel-level instrumentation to monitor system events`
- `intercepting` :point_right: `observing` to make it clear that monitoring is passive and it doesn't interfere with system calls or other kernel activities and address the concern raised by @incertum in [this comment](https://github.com/falcosecurity/falco-website/pull/1012#discussion_r1236279628)
  - I've also added `non-intrusive approach` for the same reason.
 - Other rewrites inspired by @incertum in [this comment](https://github.com/falcosecurity/falco-website/pull/1012#discussion_r1236279628)

Reply:
Yes @LucaGuerra rewrite with your inout is what we should go for https://github.com/falcosecurity/falco-website/pull/1012#discussion_r1240028542

@leogr similar to my comment above, now that we have clarified few things here can we re-audit the main Falco readme one more time and possibly adjust it with all the valuable information it should have?

Reply:
@incertum agreed :+1: 

Anyway, I'd like to create a glossary or something like that to keep track of all definitions (i.e., monitoring vs capture). wdyt?

cc @vjjmiras

Reply:
huh excellent idea love it!